packages/patternfly-4/react-core/src/components/Card/CardHeadMain.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+
+export interface CardHeadMainProps extends React.HTMLProps<HTMLDivElement> {
+  /** Content rendered inside the Card Head Main */
+  children?: React.ReactNode;
+  /** Additional classes added to the Card Head Main */
+  className?: string;
+}
+
+export const CardHeadMain: React.FunctionComponent<CardHeadMainProps> = ({
+  children = null,
+  className = '',
+  ...props
+}: CardHeadMainProps) => (
+  <div className={css('pf-c-card__head-main', className)} {...props}>
```

Comment:
Why are you suing the class name directly here?  Is it not in the stylesheet?

Reply:
No, it doesn't exist there. Read more [here](https://github.com/patternfly/patternfly-react/issues/3539#issuecomment-578488823)